src/pages/patch/patchTabs/Util.tsx

```diff
@@ -0,0 +1,62 @@
+import { SorterResult } from "antd/es/table/interface";
+import {
+  TaskResult,
+  SortDirection,
+  SortOrder,
+  TaskSortCategory,
+} from "gql/generated/types";
+
+// takes sort input from the antd table and translates into part of the query string
+export const toSortString = (
+  sorts: SorterResult<TaskResult> | SorterResult<TaskResult>[]
+) => {
+  let sortStrings: string[] = [];
+  const shortenSortOrder = (order: string) =>
+    order === "ascend" ? SortDirection.Asc : SortDirection.Desc;
+  if (Array.isArray(sorts)) {
+    sorts.forEach((sort) => {
+      const singleSortString = `${sort.columnKey},${shortenSortOrder(
+        sort.order
+      )}`;
+      sortStrings = sortStrings.concat(singleSortString);
+    });
+  } else {
+    sortStrings = sortStrings.concat(
+      `${sorts.columnKey},${shortenSortOrder(sorts.order)}`
+    );
+  }
+
+  return sortStrings.join(";");
+};
+
+// takes a sort query string and parses it into valid GQL params
+export const parseSortString = (sortQuery: string | string[]): SortOrder[] => {
```

Comment:
it would be neat to pass the result of this into useSetDefaultColumnSortOrder

Reply:
I don't see anything named that. Are you suggesting to make this a hook instead?

Reply:
@john-m-liu we originally had a [useSetColumnDefaultSortOrder](https://github.com/evergreen-ci/spruce/blob/master/src/hooks/useSetColumnDefaultSortOrder.ts#L6) that helped us set the defaultSortOrder key in the columns array. Since relying sortOrder is a cleaner solution, you can actually ignore my original comment.